CHANGELOG.md

```diff
@@ -27,15 +36,30 @@ Change Log
       - `Initializer.Config.newWallet`
       - `Initializer.Config.setViewingKeys`
 - `cash.z.ecc.android.sdk`:
-  - `Synchronizer.shieldFunds` now takes a transparent account private key (representing
-    all transparent secret keys within an account) instead of a transparent secret key.
+  - `Synchronizer.Companion.new` now takes a `seed` argument. A non-null value should be
+    provided if `Synchronizer.Companion.new` throws an error that a database migration
+    requires the wallet seed.
```

Comment:
This rewrite is insufficient. This constructor's API was completely altered due to the removal of `Initializer`, and the caller now needs to provide up-front several variables that previously they might have been ignoring and leaving to the `Initializer` defaults. The change should be described here (but it doesn't need to include a migration pathway; that can go in the migrations doc later).

Reply:
Good comments.  I will resolve this later, as the changelog will have a lot more changes with the Roomoval.